cpp/src/slice2java/JavaUtil.h

```diff
@@ -114,11 +104,7 @@ namespace Slice
         // package argument matches the entity's package name, then the
         // package is removed from the result.
         //
-        [[nodiscard]] static std::string getUnqualified(
-            const ContainedPtr&,
-            const std::string& = std::string(),
-            const std::string& = std::string(),
-            const std::string& = std::string());
```

Comment:
We no longer pass `prefix` and `suffix` in here. Instead we just add them where we need them.